cache.rst

```diff
@@ -252,6 +244,12 @@ and the other two are using the :doc:`MemcachedAdapter </components/cache/adapte
 The ``cache.acme`` pool is using the Memcached server on localhost and ``cache.foobar``
 is using the Memcached server at example.com.
 
+Each pool will manage a set of independent cache keys: keys of different pools
+never collide even if they share the same backend. This is achieved by prefixing
+keys with a namespace. This namespace is generated by hashing the name of the
+pool, the name of the compiled container class and a configurable seed that XXX HOWTO LINK TO reference/configuration/framework.rst#prefix_seed? XXX
```

Comment:
RST expert needed :)

Reply:
Will take care of this tomorrow 👌🏻

Reply:
Add this to `reference/configuration.framework.rst` file:

```diff
+ .. _reference-cache-prefix-seed

prefix_seed
...........

**type**: ``string`` **default**: ``null``

If defined, this value is used as part of the "namespace" generated for the
cache item keys. A common practice is to use the unique name of the application
(e.g. ``symfony.com``) because that prevents naming collisions when deploying
multiple applications into the same path (on different servers) that share the
same cache backend.

It's also useful when using `blue/green deployment`_ strategies and more
generally, when you need to abstract out the actual deployment directory (for
example, when warming caches offline).

```

And link it like this:
```suggestion
pool, the name of the compiled container class and a :ref:`configurable seed <reference-cache-prefix-seed>`
```